packages/components/tsconfig.json

```diff
@@ -3,7 +3,9 @@
     "compilerOptions": {
         "outDir": "./lib",
         "rootDir": "./src",
```

Comment:
@jim-deriv We also need to remove `rootDir` from here, `rootDir` will be the root of the monorepo, Otherwise we will get `ts(6059)`.

Reply:
Yeah. This is true. I realised this when I started working on `appstore`